pkg/reconciler/trigger/trigger.go

```diff
@@ -216,8 +219,9 @@ func (r *Reconciler) reconcileRetryTopicAndSubscription(ctx context.Context, tri
 	// Check if PullSub exists, and if not, create it.
 	subID := resources.GenerateRetrySubscriptionName(trig)
 	subConfig := pubsub.SubscriptionConfig{
-		Topic:  topic,
-		Labels: labels,
+		Topic:       topic,
+		Labels:      labels,
+		RetryPolicy: r.retryPolicy,
```

Comment:
Is it possible to somehow check in the unit tests that the retry policy is actually set?